docs/configuration.v2alpha1.md

```diff
@@ -249,123 +249,123 @@ In the table, `spec.override.nodeAgent.image.name` and `spec.override.nodeAgent.
 | [key].annotations `map[string]string` | Annotations provide annotations that are added to the different component (Datadog Agent, Cluster Agent, Cluster Check Runner) pods. |
-| [key].containers `map[string]object` | Configure the basic configurations for each Agent container. Valid Agent container names are: `agent`, `cluster-agent`, `init-config`, `init-volume`, `process-agent`, `seccomp-setup`, `security-agent`, `system-probe`, `trace-agent`, and `all`. Configuration under `all` applies to all configured containers. |
+| [key].containers `map[string]object` | Configure the basic configurations for each Agent container. Valid Agent container names are:`agent`, `cluster-agent`, `init-config`, `init-volume`, `process-agent`, `seccomp-setup`,`security-agent`, `system-probe`, `trace-agent`, and `all`.Configuration under `all` applies to all configured containers. |
 | [key].containers.[key].appArmorProfileName | AppArmorProfileName specifies an apparmor profile. |
 | [key].containers.[key].args `[]string` | Args allows the specification of extra args to the `Command` parameter |
 | [key].containers.[key].command `[]string` | Command allows the specification of a custom entrypoint for container |
-| [key].containers.[key].env `[]object` | Specify additional environment variables in the container. See also: https://docs.datadoghq.com/agent/kubernetes/?tab=helm#environment-variables |
-| [key].containers.[key].healthPort | HealthPort of the container for the internal liveness probe. Must be the same as the Liveness/Readiness probes. |
-| [key].containers.[key].livenessProbe.exec.command | Command is the command line to execute inside the container, the working directory for the command  is root ('/') in the container's filesystem. The command is simply exec'd, it is not run inside a shell, so traditional shell instructions ('|', etc) won't work. To use a shell, you need to explicitly call out to that shell. Exit status of 0 is treated as live/healthy and non-zero is unhealthy. |
-| [key].containers.[key].livenessProbe.failureThreshold | Minimum consecutive failures for the probe to be considered failed after having succeeded. Defaults to 3. Minimum value is 1. |
+| [key].containers.[key].env `[]object` | Specify additional environment variables in the container.See also: https://docs.datadoghq.com/agent/kubernetes/?tab=helm#environment-variables |
+| [key].containers.[key].healthPort | HealthPort of the container for the internal liveness probe.Must be the same as the Liveness/Readiness probes. |
+| [key].containers.[key].livenessProbe.exec.command | Command is the command line to execute inside the container, the working directory for thecommand  is root ('/') in the container's filesystem. The command is simply exec'd, it isnot run inside a shell, so traditional shell instructions ('|', etc) won't work. To usea shell, you need to explicitly call out to that shell.Exit status of 0 is treated as live/healthy and non-zero is unhealthy. |
```

Comment:
It seems like the spaces shouldn't have been removed here?

Reply:
Thanks! This should be fixed by 5th commit.